docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
```

Comment:
For non-pre-produced OCSP responders, short-lived delegated signing certificate should be used indeed. Currently, there are CA's that use long-lived delegated signing certificates (up to 3 years). But also here the compromise of signing certificate 'equals' the compromise of the issuing CA, only for a shorter period of time.

I agree that there is a risk of exposing any signing capable service, including for example timestamping which could be used to backdate software or documents to before the revocation date.

If we can increase the response validity allowance for revoked certificates to the end-entity certificate expiry date we can reduce the number of `non-queried` responses to be created as no reproduction is needed.
 
If there are no concerns to increase the time between response updates significantly, for example, to every 3 days instead of about every hour (except for status changes). It might be feasible to pre-produce. I'm still concerned about the request spikes caused by large sites/implementations as result.